tide_landing_page.install

```diff
@@ -287,3 +287,36 @@ function tide_landing_page_update_8004() {
     }
   }
 }
+
+/**
+ * Replace show_news_search field by Header Component Embedded Search Form [SDPA-943].
+ */
+function tide_landing_page_update_8005() {
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_landing_page') . '/config/install'];
+  _tide_import_single_config(' field.storage.paragraph.field_paragraph_search_block', $config_location);
+  _tide_import_single_config('field.field.paragraph.embedded_search_form.field_paragraph_search_block', $config_location);
+  _tide_import_single_config('field.field.node.landing_page.field_landing_page_header', $config_location);
+
+  $pages = \Drupal::entityTypeManager()->getStorage('node')
+    ->loadByProperties(['type' => 'landing_page']);
+  if ($pages && count($pages)) {
+    /** @var \Drupal\node\Entity\Node $node */
+    foreach ($pages as $node) {
+      if ($node->hasField('field_show_news_search') and $node->field_show_news_search->getValue()[0]['value'] == 1) {
```

Comment:
For consistency we should use the `&&` logical operator instead of `and`. [This discussion](https://www.drupal.org/project/coding_standards/issues/2890524) on d.o covers both coding style and technical reasons as to why it's preferable.

We should also use the strict comparison operator. See [this d.o thread for discussion](https://www.drupal.org/project/coding_standards/issues/2795691) around why.